### DISCUSSION TITLE
Tweet URL

### DISCUSSION THREAD

**Discussion by CJE90** [plhery/node-twitter-api-v2#131] [Q&A]
When grabbing a user timeline its trivial to get the ID and text, but is it possible to also get the URL for those tweets? I dont see anything in the documentation that mentions that.

**Comment by CJE90** [comment]
In hindsight, this was a very bad question.

The id given will form the URL

**Comment by alkihis** [comment]
Hello!
The tweet URL isn't returned by the API.
To make a tweet URL, you have two choices: 
- If you know the username (@) of tweet publisher (like in v1 API) you can generate the "real" link: https://twitter[.]com/{username}/status/{tweetId}
- you can also generate a generic link: https://twitter[.]com/i/statuses/{tweetId}
